helpers/inspect_files.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
I would prefer the file to be called "inspect_changed_files.sh" or "changed_files.sh" but it is not that important.